pynest/nest/lib/hl_api_info.py

```diff
@@ -200,6 +200,11 @@ def set_verbosity(level):
     - M_ERROR=30, display error messages and above
     - M_FATAL=40, display failure messages and above
 
+    .. note::
+
+       If the environment variable PYNEST_QUIET is set, NEST will not print
```

Comment:
```suggestion
       You can also set ``export PYNEST_QUIET = 1`` before importing NEST.  
   ```

Reply:
Not technically the same thing; the former is correct.

Reply:
ok, how about keep your text but also include a link to see :ref:`refguide_2_3` where the command is written